examples/integration/SanitizerApp/Source/Sanitizer app.entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Once moved to its own example, let's remove all of this integration specific stuff. The example can be as slim as needed to show off the sanitizers. So no device `target_environment`, no custom entitlements, just as bare as possible.

Reply:
Sounds good! 👍🏼